CHANGELOG.unreleased.md

```diff
@@ -11,12 +11,15 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [SDN Controller] Ensure the correct PIF is used to create private networks tunnels - Fixes [#4737](https://github.com/vatesfr/xen-orchestra/issues/4737) (PR [#4757](https://github.com/vatesfr/xen-orchestra/pull/4757))
```

Comment:
Please add `Fixes` to the `PR` description instead.

Reply:
I don't understand your comment.

Reply:
Please look `PR` description first checkbox.

Reply:
Done :)

Reply:
You can deleted from the change log entry.

Reply:
Done :)